nautobot_circuit_maintenance/tests/test_api.py

```diff
@@ -32,8 +32,8 @@ def setUpTestData(cls):
         cls.create_data = [
             {
                 "name": "UT-TEST1",
-                "start_time": now() + datetime.timedelta(days=5),
-                "end_time": now() + datetime.timedelta(days=6),
+                "start_time": utcnow() + datetime.timedelta(days=5),
+                "end_time": utcnow() + datetime.timedelta(days=6),
```

Comment:
The [Python datetime docs](https://docs.python.org/3/library/datetime.html#datetime.datetime.now) recommend using `now(timezone.utc)` instead of `utcnow()` as the former creates a timezone-aware UTC time whereas the latter creates a naive (timezone-unaware) time. We're doing our best to use aware timestamps in the actual production code for this plugin, is there a a reason not to do so in the tests as well?

Reply:
will do too